powerquery-docs/connectors/cloudbluepsa.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Power Query CloudBlue PSA connector
```

Comment:
In some cases, the connector is referred to as "CloudBlue PSA" (with a space), and other times as "CloudBluePSA" (without a space). This name needs to be consistent throughout the article.